flink-runtime/src/test/java/org/apache/flink/runtime/externalresource/ExternalResourceUtilsTest.java

```diff
@@ -162,4 +170,65 @@ public void testGetExternalResourcesWithMultipleExternalResource() {
 		assertThat(configMap.get(resourceConfigKey1), is(resourceAmount1));
 		assertThat(configMap.get(resourceConfigKey2), is(resourceAmount2));
 	}
+
+	@Test
+	public void testConstructExternalResourceDriversFromConfig() throws Exception {
+		final Configuration config = new Configuration();
+		final String driverFactoryClassName = "org.apache.flink.runtime.externalresource.TestingExternalResourceDriverFactory";
```

Comment:
Could use `TestingExternalResourceDriverFactory.class.getName()`. Same for the other occurences.